.github/workflows/pr.yml

```
@@ -60,6 +60,15 @@ jobs:
         run: yarn devtools:test
       - name: Test build
         run: yarn devtools:build:chrome
+      - name: Install Cypress
+        run: yarn global add cypress@14.4.1
+      - name: Cypress run
+        uses: cypress-io/github-action@v6
```

Actions should be pinned to a specific sha

```suggestion
        uses: cypress-io/github-action@6c143abc292aa835d827652c2ea025d098311070 # v6.10.1
```